src/backend/commands/matview.c

```diff
@@ -284,7 +285,7 @@ static void mv_HashPreparedPlan(MV_QueryKey *key, SPIPlanPtr plan);
 static void mv_BuildQueryKey(MV_QueryKey *key, Oid matview_id, int32 query_type);
 static void clean_up_IVM_hash_entry(MV_TriggerHashEntry *entry);
 static void clean_up_IVM_temptable(Oid tempOid_old, Oid tempOid_new);
-
+static List *get_securityQuals(Oid relId,Query *query);
```

Comment:
Very very little point, but I want a space from the comma.

Reply:
Sorry, I will fix it.